lib/ansible/modules/source_control/gitlab_project_variable.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Markus Bergholz (markuman@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+module: gitlab_project_variable
+short_description: Creates/updates/deletes GitLab Projects Variables
+description:
+  - When a project variable does not exists, it will be created.
+  - When a project variable does exists, its value will be updated when the values are different.
+  - Variables which are presented in the playbook, but are not presented in the Gitlab project,
+    they stay untouched (I(purged_vars) is C(false)) or will be deleted (I(purged_vars) is C(true)).
```

Comment:
What is `purged_vars`, BTW? I now saw that this is not an option for this module.

Reply:
Ah. My origin version used `purged_vars`. But I changed it because @Lunik suggested to change it just to `purge`. See https://github.com/ansible/ansible/pull/56574#discussion_r285394389  
But I forget to change it in the documentation also.